doc/start/documenting-ceph.rst

```diff
@@ -313,14 +313,14 @@ For CentOS/RHEL distributions, the remaining python packages are not available i
 the default and ``epel`` repositories. So, use http://rpmfind.net/ to find the
 packages. Then, download them from a mirror and install them. For example::
 
-	wget ftp://rpmfind.net/linux/centos/7.0.1406/os/x86_64/Packages/python-jinja2-2.7.2-2.el7.noarch.rpm
+	wget http://rpmfind.net/linux/centos/7.3.1611/os/x86_64/Packages/python-jinja2-2.7.2-2.el7.noarch.rpm
```

Comment:
It is better
I have fixed it in a new :pull request doc: update some information of document #14649
(something wrong happened when I update it.)

welcome review and comment.